docs/source/user-guide/introduction.md

```diff
@@ -106,6 +106,7 @@ Here are some active projects using DataFusion:
 - [GlareDB](https://github.com/GlareDB/glaredb) Fast SQL database for querying and analyzing distributed data.
 - [InfluxDB IOx](https://github.com/influxdata/influxdb_iox) Time Series Database
 - [Kamu](https://github.com/kamu-data/kamu-cli/) Planet-scale streaming data pipeline
+- [LanceDB](https://github.com/lancedb/lancedb) Developer-friendly, serverless vector database for AI applications
```

Comment:
Although it seems technically the `lance` (not `lancedb`) repo has the depedency 🤔  maybe we should say the lance file format rather than lancedb ?

Reply:
Yea, it makes more sense to list `lance` (direct dependency) instead of `lancedb` (transitive dependency), I think.

Reply:
Perhaps @wjones127  or @eddyxu  has a suggestion

Reply:
Thanks for including us in this document!

Yeah we can put Lance here instead, as that's the primary level we use DataFusion at.

```suggestion
- [Lance](https://github.com/lancedb/lance) Modern columnar data format for ML
```